src/Table.cpp

```diff
@@ -101,4 +101,37 @@ namespace voltdb {
             ostream << std::endl;
         }
     }
+
+    void Table::operator >> (std::ostream &ostream) const {
+
+        int32_t size = m_buffer.limit();
+        ostream.write((const char*)&size, sizeof(size));
+        if (size != 0) {
+            ostream.write(m_buffer.bytes(), size);
+        }
+    }
+
+    bool Table::operator!=(const Table& rhs) const {
+        // ToDo: Implement comparison ... byte comparison of buffer???
+        return m_buffer != rhs.m_buffer;
```

Comment:
This will only work when the two tables share the same underlying buffer. A proper approach is to implement `operator==` and compare `m_rowCount` followed by `m_buffer.bytes()`. `operator!=` can then be implemented by negating `operator==`.